### PR TITLE
feat: add resize option

### LINKED ISSUE #28: Resizing of messages pane isn't quite right
The messages pane under the editor should be resizable by dragging on the border between the editor and the pane.

Right now, you need to drag the lower-right corner of the pane to resize it. This probably means we can't rely on CSS resizing for this feature.

Screenshot 2022-04-21 at 13-44-36 ESLint Playground

### PR DESCRIPTION
**Pull request by amareshsm**
**Description:**

Added resizable split-view to the playground to resize editor and msg pane.

Fix #28 

https://user-images.githubusercontent.com/30730124/170859446-8148e471-651c-45b8-ac27-888cbcf4dc75.mov

### CODE REVIEW

**Comment by amareshsm** [comment]
#28

**Review by snitin315 — APPROVED** [review]
This looks good. Thank you for contributing.

Would like a review from @nzakas as well.

**Review by nzakas — CHANGES_REQUESTED** [review]
This looks really nice! Can you also remove `resize: vertical` from the style for that pane?

File: src/playground/scss/split-pane.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.resizable__container {
+    height: inherit;
+}
+.gutter {
+    background-color: #35415e;
+    background-color: var(--divider-color);
```
Comment (harish-sethuraman) [inline src/playground/scss/split-pane.scss:6]: Remove the one that isn't needed?

Comment (amareshsm) [reply src/playground/scss/split-pane.scss:6]: sure  👍

**Review by nzakas — APPROVED** [review]
LGTM. Thanks so much!

**Pull request merged**